tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Cpp.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Azure.Sdk.Tools.PerfAutomation.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Azure.Sdk.Tools.PerfAutomation
+{
+    // Prerequisites
+    // 1. Set VCPKG env vars
+    //    Windows: set VCPKG_ROOT=C:\Git\vcpkg & set VCPKG_DEFAULT_TRIPLET=x64-windows-static
```

Comment:
How is someone supposed to know to do these pre-reqs if they are just running the tool directly?

Reply:
These were the standard requirements for anyone working in the C++ repo.  I copied them here for my convenience.

Looking at the latest docs for C++, I don't believe these steps are required anymore, since the repo has been updated to automatically setup vcpkg:

https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#third-party-dependencies

Tracking issue: [perf-automation] Remove prerequisites from Cpp.cs comments #3459